### ISSUE TITLE
Vagrant Manager” can’t be opened because Apple cannot check it for malicious software.

### ISSUE THREAD

**Issue by royteeuwen** [lanayotech/vagrant-manager#187]
I can't open the app because of this exception. I also tried to build the software manually so that it is signed as my own user, but I get the following exception:

Showing Recent Issues
/Users/royteeuwen/Temp/vagrant-manager/Pods/Target Support Files/Pods-Vagrant Manager/Pods-Vagrant Manager.debug.xcconfig: unable to open file (in target "Vagrant Manager" in project "Vagrant Manager")

**Comment by markushausammann** [comment]
Same here, recent Catalina will not allow using it.

**Comment by SanderVanLeeuwen** [comment]
You'll get this error by default, but it's always possible to open apps anyway through the system preferences. Open your preferences panel, and go to the 'general' tab of 'security & privacy'. It'll show the last application you tried to open which gave this error, and a button to open it anyway.

If you've done this once for an app, the error will be ignored the next time.

**Comment by markushausammann** [comment]
@SanderVanLeeuwen nope, that's not the case. This is how it looks AFTER I say "open anyway":
<img width="338" alt="Security___Privacy_and_Releases_·_lanayotech_vagrant-manager" src="https://user-images.githubusercontent.com/723043/94111122-ea933000-fe43-11ea-911e-e4f0228a9669.png">

**Comment by SanderVanLeeuwen** [comment]
Yes, and then the 'open' button does open it anyway. Or at least it does in my case? Just downloaded, installed and opened it on Catalina 10.15.5.

**Comment by markushausammann** [comment]
Ok, yes. Last time around that threw another error. But why can't you let it be scanned? I don't usually install packages with so many warnings. Crosses a red line, kinda.

**Comment by SanderVanLeeuwen** [comment]
As far as I know it's because it's not properly signed with an Apple developer certificate or something alike? But don't ask me, I'm not an app developer. Just a user trying to leave a useful comment here ;)

**Comment by markushausammann** [comment]
Right, your comment WAS useful :). Sorry for confusing you with the maintainers. Where are they anyways...

**Comment by 825i** [comment]
This really needs priority. I can't use the app either or force it to run because my organisation does not allow running apps from untrusted sources. Is there anyone still alive in the dev team?
